### PR TITLE
Removing monasca from upgrade scenario

### PR DESCRIPTION
**Pull request by gosipyan**
Currently monasca failing to deploy on SOC9 CI jobs in general
removing from upgrade scenario (till fix will be provided as current
failures not related to upgrade)to have stable upgrade job running on ecp.

### CODE REVIEW

**Comment by gosipyan** [comment]
tested: https://ci.suse.de/view/Cloud/view/Cloud%209%20Crowbar/job/cloud-crowbar9-job-upgrade-x86_64/147/

File: jenkins/ci.suse.de/cloud-crowbar9.yaml [hunk 1]
```diff
@@ -54,13 +54,12 @@
...
     upgrade_cloudsource: stagingcloud9
     cloud_env: cloud-crowbar-ci-slot
-    scenario_name: std-lmm
+    scenario_name: standard
     controllers: '1'
     computes: '1'
     ses_enabled: false
-    enabled_services: database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,horizon
-    extra_params: |
-      ssl_enabled=false
```
Comment (jsuchome) [inline jenkins/ci.suse.de/cloud-crowbar9.yaml]: How is removal of this relevant to monasca removal?

Comment (gosipyan) [reply jenkins/ci.suse.de/cloud-crowbar9.yaml]: As ssl working already I decided enable it also. Can create separate PR if needed.

Comment (jsuchome) [reply jenkins/ci.suse.de/cloud-crowbar9.yaml]: No need for separate PR, but mentioning it in commit message would be nice

**Pull request merged**